gui/default/syncthing/core/syncthingController.js

```diff
@@ -554,9 +573,20 @@ angular.module('syncthing.core')
 
                 data = data.connections;
                 for (id in data) {
+                    if (!(id in $scope.idToRemoteGUIAddress)) {
+                        $scope.idToRemoteGUIAddress[id] = "";
+                    }
                     if (!data.hasOwnProperty(id)) {
                         continue;
                     }
+                    var port = $scope.findDevice(id).remoteGUIPort;
+                    var isNotRelayConnection = !data[id].type.includes("relay");
+                    if ($scope.showRemoteGUI && port !== "0" && isNotRelayConnection) {
```

Comment:
This does a string comparison for the port, yet I think it's now a number?

Reply:
I thought that was interesting too. Vanilla javascript is weird. It works though. Don't believe me. Try this in the console.
"123" + 3
output: "1233"
I will put the toString() anyway though for readability.

Reply:
Right, I missed the .toString() at the very start of creating `var port`.